ci-operator/config/openshift/openshift-tests-private/tools/add_missed_upgrade_jobs.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+"""
+This script is used to add missed e2e profiles in upgrade config file.
+***NOTE: 
+after running this script, you still need to
+1. check the new config file manually, for example, you need to update the test chain step.
+2. run 'make update'
+***
+
+Here is the example to run this script:
+```
+e2e_yaml=ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml
+upgrade_yaml=ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.13.yaml
+mode='y' # means upgrade path, default is 'y'
+
+python ci-operator/config/openshift/openshift-tests-private/tools/add_missed_jobs.py -e $e2e_yaml -u $upgrade_yaml -m "y"
```

Comment:
We may also need to update the name here ?

Reply:
done